tools/ci_build/github/azure-pipelines/templates/component-governance-component-detection-steps.yml

```diff
@@ -18,15 +18,16 @@ steps:
           and(eq('${{parameters.condition}}', 'always'), always())),
           and(eq('${{parameters.condition}}', 'succeeded'), succeeded()))
     inputs:
-      # ignore unit tests in emscripten. emscripten unit tests are not used in onnxruntime build
-      # ignore onnx third_party directory. onnx third_party directory is not responsible for onnxruntime build
-      # because sometime there are multiple repo is checked out in the same pipeline, we also need to add the repo name, such as onnxruntime/ to the ignoreDirectories
-      # ignore onnxruntime-extensions directory. onnxruntime-extensions directory is not responsible for onnxruntime build
+      # ignore cmake/external these are not our responsibility
       # ignore react_native e2e node_modules directory. react_native e2e node_modules directory is generated by react_native e2e tests
       # ignore github directory. github directory is used for github actions, not for onnxruntime package
+      # ignore onnxruntime-inference-examples directory. onnxruntime-inference-examples directory is used for inference examples, not for onnxruntime package
+      # ignore manylinux directory. manylinux is a third party directory, not responsible by onnxruntime
+      # ignore BinariesDirectory. BinariesDirectory is used for build output, not for onnxruntime package
       ignoreDirectories:
-        '$(Build.Repository.LocalPath)/cmake/external/emsdk/upstream/emscripten/tests,
-        $(Build.Repository.LocalPath)/cmake/external/onnx/third_party,
-        $(Build.Repository.LocalPath)/cmake/external/onnxruntime-extensions,
+        '$(Build.Repository.LocalPath)/cmake/external,
         $(Build.Repository.LocalPath)/js/react_native/e2e/node_modules,
-        $(Build.Repository.LocalPath)/tools/ci_build/github'
\ No newline at end of file
+        $(Build.Repository.LocalPath)/tools/ci_build/github,
```

Comment:
This is our source folder. We should not exclude it.

Reply:
This is where GHA tvm warnning coming from.

Reply:
The owner of the EP should fix it.